staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2397,6 +2398,30 @@ type SeccompConfiguration struct {
 	VirtualMachineInstanceProfile *VirtualMachineInstanceProfile `json:"virtualMachineInstanceProfile,omitempty"`
 }
 
+// WorkloadMemoryOverCommitmentProfile holds the workload memory over-commitment profile for Kubevirt.
+type WorkloadMemoryOverCommitmentProfile struct {
```

Comment:
While I like what you have done here (a struct for future expansion) I don't think the name is spot on as FPR is not technically memory over-commitment.

Maybe something like this is more appropriate?
```
type WorkloadMemoryProfile struct {
    Strategy *Strategy
    Custom *WorkloadMemoryConfiguration
}

type WorkloadMemoryConfiguration struct {
	FreePageReporting *bool `json:"freePageReporting,omitempty"`
}

```